internals/schemas/app.schema.json

```diff
@@ -73,6 +77,7 @@
         "links",
         "logo",
         "map",
+        "mapFilter24Hours",
```

Comment:
Shouldn't be required, right?

Reply:
Debatable. All boolean parameters could be seen as not-required. If they're not specified, they'll get a value of false implicitly. However, the code base requires them to have a value; they are either true or false, undefined doesn't mean anything. So in that sense, they are required. And I would rather have their values explicitly defined in the configuration than implicitly set.

Reply:
Not entire true that `undefined` doesn't mean anything. Looking at the condition

```jsx
{showsMap && configuration.mapFilter24Hours && <MapHeading>Afgelopen 24 uur</MapHeading>}
```

`mapFilter24Hours` is expected to be truthy and in javascript that can be a lot of things. Where at the same time `undefined` is not truthy. Unless you specifically check for `configuration.mapFilter24Hours === true`, it doesn't matter if the config parameter is present or not.

Reply:
That is totally true!

I was more thinking on the level of what the parameter means, represents. It's a feature flag, either on or off, true or false (truthy or falsy).
And it's different from a parameter like `matomo`, whose value should be an object which can contain different values to configure matomo. It is not required, though, and when it is not defined, matomo is not configured, and thus turned off in the application.
This makes me realize that matomo is also turned off implicitly and thus undermining my argument not allowing feature flags to be turned off implicitly.

So, we could make `mapFilter24Hours` not required, but then I would consistently make all feature flags not required (`fetchDistrictsFromBackend`, `fetchQuestionsFromBackend`, `fetchSourcesFromBackend`, `showVulaanControls`, `useStaticMapServer`).

Reply:
I also prefer to be `explicit` with the configuration options and make it required so every 'domain developer' will be aware of this option by just looking to the specific domain configuration, and if we will ever switch to strong typing it will also help.

Reply:
Okay. All required for now.
Good topic of discussion, but out of scope for this PR.

Good to think about this. What are the base settings (Amsterdam specific vs neutral). Not defined vs explicitly defined as false or null.
I just realized that sentry and matomo are enabled for other municipalities now with the change in configuration. Since these settings are not defined, because not needed. Worked fine in the past, but now it keeps the default setting of Amsterdam. So, should be explicitly overridden with null for every other deployment besides Amsterdam. Or should be undefined in the base config and Amsterdam should explicitly set these values for their deployments because Amsterdam needs sentry and matomo.